apis/catalog/v1alpha1/cassandra_version_types.go

```diff
@@ -63,9 +62,14 @@ type CassandraVersionSpec struct {
 	// Exporter Image
 	Exporter CassandraVersionExporter `json:"exporter"`
 
+	// Cassandra-medusa image
+	Medusa CassandraVersionMedusa `json:"medusa"`
+
```

Comment:
type CassandraVersionMedusa struct {
	Image string                     `json:"image"`
	Init  CassandraVersionMedusaInit `json:"init"`
}

type CassandraVersionMedusaInit struct {
	Image string `json:"image"`
}

Add only one field for medusa in version.spec